### DISCUSSION TITLE
Fonts and visual appearance of chat

### DISCUSSION THREAD

**Discussion by sibawal** [innightwolfsleep/llm_telegram_bot#141] [General]
People, is it possible to somehow make the font and color the same as in a regular telegram chat?
Now it looks like this

![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/52999882/cc5ca531-3d16-4a7e-940b-49c77aa95189)

But I would like to make it simpler, like regular correspondence.
![image](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/assets/52999882/2b11188c-87e6-46f9-ad3e-068dbcf03721)

Where are the font settings?

**Comment by innightwolfsleep** [comment]
Понял что пропустил вопрос... Наверно уже не актуально, но:
Forgot about question... Perhaps its too late, but answer is: 
[conf.py](https://github.com/innightwolfsleep/text-generation-webui-telegram_bot/blob/main/source/conf.py)
```
    html_tag = Field(default=["<pre>", "</pre>"], description="html tags for ordinary text")
    translate_html_tag = Field(
        default=['<span class="tg-spoiler">', "</span>"], description="html tags for translated text"
    )
```
